tests/snapshots/files__fib_shape-rvsdg-optimize.snap

```diff
@@ -0,0 +1,220 @@
+---
+source: tests/files.rs
+expression: visualization.result
+---
+@main(v0: int) {
+  v1: int = const 0;
+  v2: int = const 1;
+  v3: int = id v1;
+  v4: int = id v2;
+  v5: int = id v0;
+  jmp .__7__;
+.__130__:
+  v128: int = const 1;
+  v129: bool = eq v121 v128;
+  v68: int = id v87;
```

Comment:
(no change needed): this is what they mean by "copy insertion" in the "Perfect Reconstruction" paper. 

Sounds like there are standard algorithms here to clean this up.

Reply:
ohhhh makes more sense to me now